src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -203,16 +201,11 @@ public String getName() {
      */
     @Nullable
```

Comment:
It is still nullable, as the called createSession is also nullable. (on null user and password)